core/templates/dev/head/pages/preferences-page/preferences-page.controller.spec.ts

```diff
@@ -71,6 +71,7 @@ describe('Preferences Controller', function() {
         $rootScope: $rootScope,
         AlertsService: mockAlertsService
       });
+      ctrl.$onInit();
```

Comment:
Explanation for this change: https://www.codelord.net/2017/01/09/unit-testing-angular-components-with-%24componentcontroller/

Reply:
Sure, will add once this PR goes in.